tests/05synapse.pl

```diff
@@ -79,24 +79,32 @@ END
 
          if( $idx == 0 ) {
             # Configure application services on first instance only
-            my $appserv_conf = $synapse->write_yaml_file( "appserv.yaml", {
-               url      => $test_server_info->client_location . $as_user_info->path,
-               as_token => $as_user_info->as2hs_token,
-               hs_token => $as_user_info->hs2as_token,
-               sender_localpart => $as_user_info->localpart,
-               namespaces => {
-                  users => [
-                     { regex => '@astest-.*', exclusive => "true" },
-                  ],
-                  aliases => [
-                     { regex => '#astest-.*', exclusive => "true" },
-                  ],
-                  rooms => [],
-               }
-            } );
+            my @confs;
+
+            foreach my $idx ( 0 .. $#as_infos ) {
+               my $as_info = $as_infos[$idx];
+
+               my $appserv_conf = $synapse->write_yaml_file( "appserv-$idx.yaml", {
+                  url      => $test_server_info->client_location . $as_info->path,
+                  as_token => $as_info->as2hs_token,
```

Comment:
Ah, but it is in the source code for that file. How odd. Maybe the site hasn't been pushed lately.